vale/vocab.txt

```diff
@@ -0,0 +1,137 @@
+addr
+api
+apis
+architecting
+artifactory
+Artifactory
+Atlassian
+autogenerated
+autoscaling
+Autoscaling
+Basecamp
+Basscss
+Bazel
+Bitium
+boolean
+boomper
+Boomper
+Buildbox
+buildkite
+Buildkite
+buildscript
+chatbot
+chroot
+chroots
+cli
+cloudformation
+Cloudwatch
+composable
+config
+cpu
+cron
+crontab
+Crontab
+Datadog
+Dockerfile
+dogstatsd
+env
+eval
+excercised
+Explainshell
+fff
+ffxdq
+Firewalled
+Flexbox
+flickr
+gcloud
+gif
+github
+Github
+Gitlab
+globbing
+golang
+Gradle
+gz
+Hashicorp
+heroku
+hostname
+http
+https
```

Comment:
Are a lot of these entries from URLs? Or just code blocks? I see a lot of very lowercase words I wouldn't expect to turn up in prose.